packages/linux-driver-addons/dvb/crazycat/patches/amlogic/driver.dvb.crazycat-04-AML-hacks.patch

```diff
@@ -11,6 +11,10 @@
 +	# Copy videobuf-res module
 +	cp -a "$(SRCDIR)/drivers/media/v4l2-core/videobuf-res.c" "drivers/media/v4l2-core/"
 +	cp -a "$(SRCDIR)/include/media/videobuf-res.h" "include/media/"
++	mkdir -p  drivers/media/dvb-avl/
```

Comment:
could we add here an `# addds support for driver abc for device ...` comment ?

Reply:
I can provide in comments my solution for wetekdvb as an example.